### PR TITLE
FUSETOOLS-3009 - replace Fuse Ignite with Fuse Online in UI

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [x] Are all Sonar reported issues fixed or can they be ignored?
- [x] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by lhein** [comment]
this PR should go into 11.1

File: configuration/ReadMe.md [hunk 1]
```diff
@@ -61,15 +61,15 @@ This file is currently used by older versions of the tooling to map versions for
 
 
 ## igniteVersionToDisplayName.properties
-This file maps the Fuse Ignite version to a human readable name to be displayed in the New Fuse Ignite Wizard. It replaces the old ignite.properties.
+This file maps the Fuse Online version to a human readable name to be displayed in the New Fuse Online Extension Wizard. It replaces the old ignite.properties.
 
 Syntax:
 
-	<ignite.version>=<human readable name of this version>
+	<fuse.online.version>=<human readable name of this version>
```
Comment (apupier) [inline configuration/ReadMe.md]: the online file should be changed also? Or need to use another one to keep backward-compatibility?

Comment (lhein) [reply configuration/ReadMe.md]: I don't get your question. That is just a key name for the readme. The config file only contains versions.

Comment (apupier) [reply configuration/ReadMe.md]: I don't understand to what this key is referring then. Where is it used?

Comment (lhein) [reply configuration/ReadMe.md]: It is just an explanation on how to fill in values to the config file. Its like <key>=<value> which just tries to make it easy for users to fill in the correct values.

Comment (apupier) [reply configuration/ReadMe.md]: wow I just get it, I thought the ignite.version or fuse.online.version was used directly as a key somewhere. My bad

**Pull request merged**